### DISCUSSION TITLE
Email notifications Issue

### DISCUSSION THREAD

**Discussion by jbryanmiller** [DefectDojo/django-DefectDojo#5011] [General]
I am trying to setup email notifications and am seeing "Must issue a STARTTLS". I am trying to use it with aws ses, i have tried ports 25 and 587 with same error. Has anyone gotten this work before ? Any help would be greatly appreciated. 
Currently using version 2.0.3

**Comment by valentijnscholten** [comment]
Can you check https://app.slack.com/client/T04T40NHX/C2P5BA8MN/thread/C2P5BA8MN-1631796429.185200 and https://github.com/DefectDojo/django-DefectDojo/issues/5112
